include/zephyr/dt-bindings/input/input_key_code.h

```diff
@@ -0,0 +1,753 @@
+/*
+ * Copyright (c) 2022, tangchunhui@coros.com
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_INPUT_INPUT_KEY_CODE_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_INPUT_INPUT_KEY_CODE_H_
+
+/*
```

Comment:
This is simply copied from [linux-event-codes.h](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/include/uapi/linux/input-event-codes.h?h=v5.19#n64) without any attribution. Note that the linux header is licensed under `GPL-2.0-only WITH Linux-syscall-note`